src/interactivity/base/InteractivityFactory.cpp

```diff
@@ -479,12 +512,12 @@ using namespace Microsoft::Console::Interactivity;
     }
     case WM_ACTIVATE:
     {
-        if (const auto ownerHwnd{ ::GetAncestor(hWnd, GA_ROOTOWNER) })
+        if (const auto ownerHwnd = _owner.load(std::memory_order_relaxed))

```

Comment:
The benefit of this change is that we now assign focus to the exact HWND that we were told is our owning window and not any window up the `GA_ROOTOWNER` chain.

Reply:
Oh, this is great